docs/sphinx/source/installation.rst

```diff
@@ -38,23 +56,30 @@ For best compatibility, deploy in an `Anaconda 2.7` environment, or run::
 
 STEP 2
 ~~~~~~
-Move gencumulativesky.exe
+Windows:
 
 * Copy gencumulativesky.exe from the repo's ``/bifacial_radiance/data/`` directory and copy into your Radiance install directory.
   This is typically found in ``/program files/radiance/bin/``.  
  
+Linux/Mac OSX:
+
+* Copy the gencumulativesky executable from the repo's ``/bifacial_radiance/data/`` directory and copy into your Radiance install directory.
+  This is typically found in ``/usr/local/radiance/bin/``. 
+* For Linux/Mac OSX, you will need to install QT for the GUI to work properly. Installation and details described in (:issue:`130`:).
```

Comment:
Personally, I think it is cleaner to add those instructions here and close that issue (which is #131 and not #130, by the way.)

Reply:
added the detailed instructions to the pre-requisite section. Left the reference to the issue 131 since I have no way of testing this, or in case people identify some other issues while installino.